bowtie/tests/test_benchmarks.py

```diff
@@ -53,6 +53,26 @@ def invalid_benchmark():
     return invalid_benchmark.get_benchmark()
 
 
+@pytest.fixture()
+def benchmark_report1():
```

Comment:
Fixtures should be for "obviously" reusable things. Here (and in the existing tests) you basically create fragility more than anything -- because it's not clear what someone could use these fixtures for *other* than the very specific test you wrote for merging -- that means you should just do this inline rather than making it "seem" reusable.